### ISSUE TITLE
Validation triggering immediately with placeholder attribute

### ISSUE THREAD

**Issue by JeffEmery** [formvalidation/support#389]
When a validated input has a `placeholder` attribute, the is validated on focus giving the error feedback immediately.

This code works as expected...
```html
<div class="form-group">
  <label for="FirstName" class="control-label">First Name</label>
  <input type="text" name="FirstName" class="form-control" />
</div>
<div class="form-group">
  <label for="LasttName" class="control-label">First Name</label>
  <input type="text" name="LastName" class="form-control" />
</div>
```

This code demonstrates the bad behavior...
```html
<div class="form-group">
  <label for="FirstName" class="control-label">First Name</label>
  <input type="text" name="FirstName" class="form-control" placeholder="First Name" />
</div>
<div class="form-group">
  <label for="LasttName" class="control-label">First Name</label>
  <input type="text" name="LastName" class="form-control" placeholder="Last Name" />
</div>
```

**Comment by nghuuphuoc** [comment]
Can you tell me the version of FormValidation you're using?

**Comment by JeffEmery** [comment]
Just downloaded 0.7.0 a couple days ago.

**Comment by nghuuphuoc** [comment]
One more question, does it work on Internet Explorer 9 and 10? (Since this issue seems to be fixed on IE 9/10, so I need your confirmation too)

**Comment by JeffEmery** [comment]
I don't have an IE 9/10 install right now... :( I will try to find a way to test those versions tomorrow.

**Comment by thiru5216** [comment]
Yes, I am getting the same issue with IE 11.. and I tested on IE8-9 as well , had the same  issue...

**Comment by thiru5216** [comment]
just use the following in $().formvalidation().. i got the result... just have a try...
live:'submitted', //<-- only validate when form submitted
live:'disabled', //<-- Disable the live validation

thank you..

**Comment by vdeSolucom** [comment]
Hi,
I am getting the same issue with v0.7.1 I downloaded today.
In fact, I have 2 input fields whith placholder, if I focus the first and then click on second without typing value, I went on an infinite loop beetween the two fields :-(
I tested it on IE 10 and IE 11 and issue is on both version...
Do you have a fix for that ?
